### PR TITLE
Refund list view fails if CreditMemo entity has been overridden

### PR DESCRIPTION
**Pull request by luca-rath**

### CODE REVIEW

File: src/Resources/config/app/config.yml [hunk 1]
```diff
@@ -62,7 +62,7 @@ sylius_grid:
             driver:
                 name: doctrine/orm
                 options:
-                    class: Sylius\RefundPlugin\Entity\CreditMemo
+                    class: 'expr:parameter("sylius_refund.model.credit_memo.class")'
```
Comment (lchrusciel) [inline src/Resources/config/app/config.yml:65]:
```suggestion
                    class: %sylius_refund.model.credit_memo.class%
```

File: src/Resources/config/app/config.yml [hunk 2]
```diff
@@ -62,7 +62,7 @@ sylius_grid:
             driver:
                 name: doctrine/orm
                 options:
-                    class: Sylius\RefundPlugin\Entity\CreditMemo
+                    class: %sylius_refund.model.credit_memo.class%
```
Comment (Nek-) [inline src/Resources/config/app/config.yml:65]: AFAIK this parameter is never set inside the plugin.

Comment (luca-rath) [reply src/Resources/config/app/config.yml:65]: I guess it's the SyliusResourceBundle, which sets this parameter, but I'm not sure about that. But the parameter is definitely available in the container.

Comment (Nek-) [reply src/Resources/config/app/config.yml:65]: I guess no? https://github.com/Sylius/SyliusResourceBundle/search?q=sylius_refund.model.credit_memo.class&unscoped_q=sylius_refund.model.credit_memo.class

Comment (luca-rath) [reply src/Resources/config/app/config.yml:65]: Magic happens here https://github.com/Sylius/SyliusResourceBundle/blob/3464579ed05b199fcd8bc12e5444b36d65d6a4a5/src/Bundle/DependencyInjection/Driver/AbstractDriver.php#L48 🙂

**Review by Zales0123 — APPROVED** [review]

**Comment by Zales0123** [comment]
Thank you, Luca, and sorry for makinging it that long 🖖

**Comment by luca-rath** [comment]
No problem, mostly a little reminder is all it takes 🙂 thanks for merging!

**Pull request merged**